src/backends/automate.js

```diff
@@ -63,7 +63,7 @@ const BROWSERSTACK_API_PATHS = {
 function requestApi (path, params) {
     return requestApiBase(path, params)
         .then(response => {
-            if (response.status !== 0)
+            if (response.status)
```

Comment:
Hi I think I made better way to fix Firefox issue in https://github.com/DevExpress/testcafe-browser-provider-browserstack/pull/56/commits/bba0fd9f17177a607b042e210aec22c8f70f3238

Reply:
I guess if some browser driver returned `status:null` or `status:''`, it would still indicate a non-error status. So such a general check is better here.

Reply:
So you are saying that status is always error?

Reply:
BTW it must be a number by the specs: https://github.com/SeleniumHQ/selenium/wiki/JsonWireProtocol#responses

Reply:
Ok. Your solution is better 👍

Reply:
Nope, I just feel that some other non-conforming web driver implementation can also return `null` and `''` as easily as Firefox returns `undefined`.